tools/translate_yaml.py

```diff
@@ -308,7 +308,7 @@ def append_products(products, idspace, outfile):
 
 
 ontobee = 'http://www.ontobee.org/browser/rdf.php?o=%s&iri=http://purl.obolibrary.org/obo/%s_$1'
-ols = 'https://www.ebi.ac.uk/ols/ontologies/%s/terms?iri=http%%3A%%2F%%2Fpurl.obolibrary.org%%2Fobo%%2F%s_$1'
+ols = 'https://www.ebi.ac.uk/ols4/ontologies/%s/entities/http%%253A%%252F%%252Fpurl.obolibrary.org%%252Fobo%%252F$1'
```

Comment:
A bit strange that the URL encoding pattern of the entity PURL changed

Reply:
@udp can you take a look at this?

CI is complaining that

```
    replacement = (ols % (idspace.lower(), idspace))
TypeError: not all arguments converted during string formatting
```

But I don't get why - also I am not clear why there is variation in the way the URI string is encoded..

Reply:
```suggestion
ols = 'https://www.ebi.ac.uk/ols4/ontologies/%s/entities/http%%253A%%252F%%252Fpurl.obolibrary.org%%252Fobo%%252F%s'
```

Reply:
Its true that this string does not have two placeholder %s to replace.

Reply:
Fixed!